csm/conf/reset.py

```diff
@@ -168,4 +168,6 @@ def _delete_cortxusers_from_ldap(self):
             raise CsmSetupError("Failed to fetch LDAP root user")
         if not self._ldappasswd:
             raise CsmSetupError("Failed to fetch LDAP root user password")
```

Comment:
Failed to fetch LDAP root user password -> Failed to fetch credentials for ldap operations

Reply:
done